src/lib/features/snackbar/model/store.ts

```diff
@@ -0,0 +1,47 @@
+import { writable, get } from 'svelte/store';
+import type { SnackbarOptions } from './types';
+
+/**
+ * The queue of different snackbars. It works like this:
+ *
+ * • First comes served – the first snackbar stays on the screen for 4 seconds
```

Comment:
nitpick (non-blocking): These bullet-point characters are neat, but JSDoc is Markdown, so they won't be rendered into proper HTML lists. In this way, it's better to use dashes `-`